src/screens/ProfileScreen/elements/BioAndDMSetting.tsx

```diff
@@ -8,8 +8,10 @@ import {COLORS} from '../../../utils/theme';
 import {Divider} from '../../../components/Divider';
 import {PencilIcon} from '../../../assets';
 import {TextWithEmoji} from './TextWithEmoji';
+import {fonts} from '../../../utils/fonts';
 import {addDotAndRemoveNewline} from '../../../utils/string/TrimString';
 import {profileSettingsDMpermission} from '../../../service/profile';
+import {fonts} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There is a redundant import statement for `fonts` from `'../../../utils/fonts'`. This import is declared twice which is unnecessary and can lead to confusion. Please remove the duplicate import.
```diff
- import {fonts} from '../../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->